android/sdk/src/main/java/com/taobao/weex/utils/WXSoInstallMgrSdk.java

```diff
@@ -242,6 +261,12 @@ public static void copyStartUpSo() {
         } else {
           WXEnvironment.extractSo();
         }
+        if(!versionFile.exists())
+          versionFile.createNewFile();
+        FileWriter fileWriter = new FileWriter(versionFile);
+        fileWriter.write(String.valueOf(WXEnvironment.CORE_JSB_SO_VERSION));
+        fileWriter.flush();
```

Comment:
Add a more try catch finally clause to close the writer as if there is a exception in line 268, fileWriter would be still open.